ocs_ci/utility/powernodes.py

```diff
@@ -29,6 +30,10 @@ def __init__(self):
             self.isKVM = True
         else:
             self.isKVM = False
+
+        if "bastion_ip" in config.ENV_DATA:
```

Comment:
This looks hackish, Is there a better method? there is already a check before is that failing?

Reply:
@vasukulkarni , bastion_ip in ENV_DATA is optional. That's why I added this check to determine if this is external ocs-ci or local ocs-ci execution.